client/src/components/Login.js

```diff
@@ -3,39 +3,56 @@ import { Link, useNavigate } from "react-router-dom";
 import axios from 'axios';
 import { useUser } from "../contexts/UserProvider";
 import { BiArrowBack } from "react-icons/bi";
+import Alert from 'react-bootstrap/Alert';
+
 
 function Login() {
 	const [credentials, setCredentials] = useState({ email: "", password: "" });
-	const navigate = useNavigate();
+	const [show, setShow] = useState(false);
+	const navigate = useNavigate(); 
 	const { setUser } = useUser();
 
 	const handleSubmit = async (e) => {
 		e.preventDefault();
 
 		try {
-			const response = await axios.post(process.env.REACT_APP_SERVER_URL + "auth", credentials)
-			const { success, token, user } = response.data;
-
-			if (!success) return alert("Invalid credentials!");
+			const response = await axios.post(`${process.env.REACT_APP_SERVER_URL}auth`, {
```

Comment:
don't pass email and password separately. Pass the whole credentials object instead. No need to write more code for doing the exact same thing.